README.md

```diff
@@ -5,65 +5,62 @@ Stores all the manifests and corresponding image-infos to test OSbuild.
 
 ### Update workflow
 
-The update workflow is made of 3 distinct steps:
+#### 1/ on GitHub: trigger the Gitlab CI
 
-#### First step, triggering the CI
+The update workflow creates a branch that is pushed to [this GitLab
```

Comment:
This is out-of-date, we don't create a branch anymore.

Reply:
Oh my! I've completely forgot the README!